src/StringHelper.php

```diff
@@ -577,4 +577,54 @@ static function (string $key) use ($delimiter, $escapeCharacter): string {
             $result
         );
     }
+
+    /**
+     * Strip Unicode whitespace (with property White_Space=yes) or other characters from the beginning and end of a string.
+     * Input string and pattern are treated as UTF-8.
+     *
+     * @see https://en.wikipedia.org/wiki/Whitespace_character#Unicode
+     * @see https://www.php.net/manual/function.preg-replace
+     *
+     * @param string $string The input string.
+     * @param string $pattern PCRE regex pattern to search for, as a string. Quote $pattern if it contains special regular expression characters.
+     * @see https://www.php.net/manual/function.preg-quote.php
+     *
+     * @return string
+     */
+    public static function trim(string $string, string $pattern = "\pC\pZ"): string
```

Comment:
```suggestion
    public static function trim(string|array $string, string $pattern = "\pC\pZ"): string
```

How about to support also a list of strings?